randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -1188,9 +1188,14 @@ Extra - room_id: [38]
       All of the following:
           Speed Booster
           Any of the following:
-              Wave Beam or Can Use Bombs
-              # Shinespark directly from adjacent room to the item
-              Screw Attack and Varia Suit and Shinespark Tricks (Expert) and Disabled Door Lock Rando and Disabled Entrance Rando and Can Use Power Bombs
+              Can Use Bombs
+              # Wave Beam is difficult to control, but can solve the puzzle (Wide Beam could be turned off if also collected)
+              Wave Beam and Knowledge (Advanced)
```

Comment:
I'm on the fence of advanced vs intermediate. but because of the weird nature of wave beam and this being a "soft-lock" of sorts I am happy to leave it at advanced.